runtimebp/config.go

```diff
@@ -0,0 +1,38 @@
+package runtimebp
+
+import (
+	"fmt"
+)
+
+// Config is the configuration struct for the runtimebp package.
+//
+// Can be parsed from YAML.
+type Config struct {
+	// NumProcesses can be used to set the maximum and minimum number of Go
+	// Processes.
+	NumProcesses struct {
+		// Max controls the maximum number of Go processes.
+		//
+		// Defaults to 50 if not set.
+		Max int `yaml:"max"`
+
+		// Max controls the minimum number of Go processes.
+		//
+		// Defaults to 1 if not set.
+		Min int `yaml:"min"`
+	} `yaml:"numProcesses"`
+}
+
+// InitFromConfig sets GOMAXPROCS using the given config.
+func InitFromConfig(cfg Config) {
+	max := 50
```

Comment:
I'm not sure if this _actually_ makes sense as a default 😅

Reply:
I think Amazon EC2 allows maximum 64 cpu, so this might be low for some cases but should be fine for a default value :)

Reply:
64 also seams reasonable

Reply:
We are currently using cpu*2-1 as the formula, so if they configure a very heavy service using all 64 cpu that would be 127 😅

Reply:
This is a ceiling though IIRC? the actual value is based on the CPUs available right?

Reply:
Yea this is the ceiling. The main purpose is to prevent some bug gives you some crazily high value.